src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -188,7 +188,7 @@ private CommandResult executeCommand(String commandText) throws CommandException
 
             return commandResult;
         } catch (CommandException | ParseException e) {
-            logger.info("Invalid command: " + commandText);
+            logger.info("An error occured while executing this command: " + commandText);
```

Comment:
Should this change be in a separate commit, as it is not related to other changes in this commit?